open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -2416,46 +3072,44 @@ PROCEDURE ERRORHANDLER
 **  509 - Error getting new share rate
 **  510 - Could not calculate new maturity date
 **  511 - Error creating new share
-**  512 - Error creating new name record(s)
-**  513 - Transfer funding error
 *]
- ERRORMSG="loggingErrorMessage"
- IF ERRORCODE="500" THEN
-  ERRORDETAIL="Program running in memo mode"
- ELSE IF ERRORCODE="501" THEN
-  ERRORDETAIL="CFG data file ("+CONFIGFILENAME+") read error: "+CFGFILEREADERROR
- ELSE IF ERRORCODE="502" THEN
-  ERRORDETAIL=FORMAT("Account Warning 999 Exists",I)
- ELSE IF ERRORCODE="503" THEN
-  ERRORDETAIL=FORMAT("Ineligible Acct Type 9999 found",ACCOUNT:TYPE)
- ELSE IF ERRORCODE="504" THEN
-  ERRORDETAIL="No eligible shares available to be created"
- ELSE IF ERRORCODE="505" THEN
-  ERRORDETAIL=FORMAT("Error reading Share group 99 term file:"+TMPCHR,VALUE(OPENCATEGORY))
- ELSE IF ERRORCODE="506" THEN
-  ERRORDETAIL="No shares with sufficient funds for required transfer"
- ELSE IF ERRORCODE="507" THEN
-  ERRORDETAIL="Error reading passed name info:"+PASSEDINFOREADERROR
- ELSE IF ERRORCODE="508" THEN
-  ERRORDETAIL="Next ID calculation error"
- ELSE IF ERRORCODE="509" THEN
-  ERRORDETAIL="Error getting new share rate"
- ELSE IF ERRORCODE="510" THEN
-  ERRORDETAIL="Could not calculate new maturity date"
- ELSE IF ERRORCODE="511" THEN
-  ERRORDETAIL="Error creating new share:"+SHAREERRORMESSAGE
- ELSE IF ERRORCODE="512" THEN
-  ERRORDETAIL="Error creating new name record(s):"+TMPNAMEERROR
- ELSE IF ERRORCODE="513" THEN
-  ERRORDETAIL="Transfer funding error:"+TRANERRORTEXT
- CALL CUERRORHANDLING
 
- ERRORCODE="500" [Per new JSON Specs, force all errors to "500"]
+ IF VALUE(ERRORCODE)>=500 AND
+    VALUE(ERRORCODE)<=MAXERRORCODE THEN
+  DO
+   TMPNUM=VALUE(ERRORCODE)-500
+   IF TESTERRORCODE>0 THEN
+    ERRORDETAIL="TESTING: "
+   ERRORDETAIL=ERRORDETAIL+ERRORCODEDESCR(ECDEFAULT,TMPNUM)
+
+   IF VALUE(ERRORCODE)=501 THEN
+    ERRORDETAIL=ERRORDETAIL+" - "+CONFIGFILENAME+": "+CFGFILEREADERROR
+   ELSE IF VALUE(ERRORCODE)=502 THEN
+    ERRORDETAIL=ERRORDETAIL+" - "+FORMAT("warning code 999",I)
+   ELSE IF VALUE(ERRORCODE)=503 THEN
+    ERRORDETAIL=ERRORDETAIL+" - "+FORMAT("Acct type 9999",ACCOUNT:TYPE)
+   ELSE IF VALUE(ERRORCODE)=505 THEN
+    ERRORDETAIL=ERRORDETAIL+" - "+FORMAT("Share Grp 99 ",VALUE(OPENCATEGORY))+LFERROR
```

Comment:
I think this error was more explicit before when it had 'term file:"+TMPCHR', consider adding it back unless you think that was redundant or would make the error too long.